sections/preliminaries.tex

```diff
@@ -0,0 +1,136 @@
+
+\begin{figure*}[ht]
+	\centering
+	\begin{subfigure}{0.275\textwidth}
+		\centering
+		\resizebox{\textwidth}{!}{%
+			\input{figures/author_plate_original}
+		}
+		\caption{Author \gls{lda} from \cite{author_topic_2012}.}
+		\label{fig:original_author_lda}
+	\end{subfigure}
+	\hspace{2em}
+	\begin{subfigure}{0.275\textwidth}
+		\centering
+		\resizebox{\textwidth}{!}{%
+			\input{figures/category_plate}
+		}
+		\caption{Category \gls{lda}.}
+		\label{fig:category_lda}
+	\end{subfigure}
+	\hspace{2em}
+	\begin{subfigure}{0.275\textwidth}
+		\centering
+		\resizebox{\textwidth}{!}{%
+			\input{figures/author_plate}
+		}
+		\caption{Author \gls{lda}.}
+		\label{fig:author_lda}
+	\end{subfigure}	
+	\caption{Plate notation for the metadata \gls{lda} models.}
+	\label{fig:metadata_lda}
+\end{figure*}
+
+\subsection{\acrlong{pam}}\label{subsec:pachinko_prelim}
+Pachinko allocation\cite{li2006pachinko} is a topic model that generalizes \gls{lda}, making it possible to construct topic hierarchies based on any \gls{dag} structure.
+The model focuses on finding topics of different abstraction levels, as well as modeling the connections between these topics.
+
+Each node in the \gls{dag} structure represents a topic in the pachinko allocation model. 
+However, unlike \gls{lda} where topics are distributions over words, in \gls{pam} topics are distributions over words and/or other topics further down the hierarchy of the \gls{dag} structure.
+
+\citet{li2006pachinko} present the four-level \gls{pam}, which is a layered \gls{pam} meaning that the \gls{dag} structure is divided into layers, with every node in each layer fully connected to every node in the next layer of the hierarchy.
+The first layer consists of only the root node, a topic which all documents are part of.
+The bottom layer consists of leaf nodes, which are the only nodes to contain distributions over words, and they have no other topics in their distribution.
+The rest are middle layers representing topics of different abstraction levels.
+
+The generative process for each document $d$ in \gls{pam} consists of the following steps, as described by \citet{li2006pachinko}:
+\begin{enumerate}
+	\item Sample $\theta_{t_1}^{(d)}, \theta_{t_2}^{(d)}, \dots, \theta_{t_s}^{(d)}$ from \vejleder{what is a g function?} $g_1(\alpha_1), g_2(\alpha_2), \dots, \newline g_s(\alpha_s)$, where $\theta_{t_i}^{(d)}$ is a multinomial distribution of topic $t_i$ over its children.
+	\item For each word $w \in d$,
+	\begin{itemize}
+		\item Sample a topic path $\mathbf{z}_w$ of length $L_w:~< z_{w1}, z_{w2},\newline \dots, z_{wL_w} >$. $z_{w1}$ is always the root and $z_{w1}$ through $z_{wL_w}$ are topic nodes in $T$. $z_{wi}$\vejleder{DAG over topics?} is a child of $z_{w(i-1)}$ and it is sampled according to the multinomial distribution $\theta_{z_{wL_w}}^{(d)}$.
+		\item Sample word $w$ from $\theta_{z_{wL_w}}^{(d)}$.
+	\end{itemize}
+\end{enumerate}
```

Comment:
Er det hele up to date med plate notation?

Reply:
dette er nærmest ordret hvad de gør.